### ISSUE TITLE
Design Call #28

### ISSUE THREAD

**Issue by Zaxounette** [btcpayserver/organization#80]
```meta
Date: 2022-02-03
Time: 5:00pm
Timezone: UTC
Duration: 1h
UTCTime: 2022-02-03 17:00 UTC
```

### Agenda

Comment below to add items to the agenda.

### Check your timezone

https://everytimezone.com/s/f7523cc4

### Join the call

[LINK FOR CALL](https://meet.jit.si/DesignCallsROCK)

### Calendar invite

Subscribe to the BTCPay Server calendar. More info [here](https://github.com/btcpayserver/organization#calendar).

**Comment by dstrukt** [comment]
Proposed Agenda:

Existing PRs:
- Any pressing PRs or feedback on existing work (@dennisreimann)

Discussion / Planning:
- 1.5.0 ....
- Updated Wallet Views & Flow Discussion
- Early Dashboard Wireframe Discussion
- Payment Method / Settings (Store Settings / Wallet / Lightning) Configuration & Consolidation Discussion
- Lightning Node View Discussion

- Updated Create Views...?

**Comment by dstrukt** [comment]
Dropping this so it's seen, and to document as we go.

1.5.0 Roadmap Potentials

Categorized with Key
Unlabeled, S, M, L, XL = workload from my POV

Dashboard:
[XL] Store Dashboard V1.0 ... Global Dashboard V1.0 ?

Checkout Invoice: 
[XL] Checkout Invoice V2.0

BTC Wallet:
[L] Wallet View Update (will include requested Balance)
RBF, CPFP Transaction View Actions (https://github.com/btcpayserver/btcpayserver/issues/2509)

Payment Requests:
[M] Improve Payment Request Flow
[M] Incorporate customer into Invoices & Payment Requests flow

Lightning:
[M] Lightning Node View Update

Pull Payments:
Add description to Pull Payments (https://github.com/btcpayserver/btcpayserver/pull/3363)

Apps:
[M] Front-End Point of Sale Improvements
[M] Front-End Crowdfund Improvements

Plugins:
[M] Update Plugin View (Low Priority)

Invoices:
[M] Improve Invoice Actions, Tags & Filtering (https://github.com/btcpayserver/btcpayserver/issues/1809)

Notifications:
[M] Improve Notification Icons + View

General:
[M-L] Fixed Headers (https://github.com/btcpayserver/btcpayserver/issues/3344)
[M-L] "Payment Method / Settings" Continued Consolidation
[M-L] Design System Improvements (see existing Issue with list)

[M] Refactor Authorize UI (https://github.com/btcpayserver/btcpayserver/issues/2902)
[M] Add "Customer"?
[M] Recovery Phrase View Improvements

[S] Improve 2FA View (Visual)
[S] Add icons to Store Selector component
[S] Improve empty state copy (https://github.com/btcpayserver/btcpayserver/issues/3266)
Remove payment methods not currently configured when creating invoice, etc.. (https://github.com/btcpayserver/btcpayserver/issues/3380)
Webhook Requests/Deliveries UI (https://github.com/btcpayserver/btcpayserver/issues/2549)
Currency Dropdown Improvements (https://github.com/btcpayserver/btcpayserver/issues/3349)

Potential:
Refunds?

**Comment by Zaxounette** [comment]
Design Call N°28
Attendees: @dstrukt @dennisreimann @pavlenex @Zaxounette 

----

From Dennis and Dstrukt's sides, nothing left is pressing.
It's now prioritizing work on what we see getting included into 1.5.0.

High level prioritization
* Updated Wallet Views & Flow Discussion
* Early Dashboard Wireframe Discussion
* Payment Method / Settings (Store Settings / Wallet / Lightning) Configuration & Consolidation Discussion (Live)
* Lightning Node View Discussion
* Checkout Invoice Discussion

----

**Updated Wallet Views & Flow Discussion**
* What happens when you go to a wallet that is deeper than what is accessible in the left menu?
  * Multiple solutions used currently: Modal, full page
  * Possible additions to the full page views: Add bread-crumbs
  * Modals work for flows where you have a context change, like wallet setup flows, but doesn't work for receive flows which are the same context as the wallet view
  * For the wallet, we have the potential to not go 2 clicks deep, but have the ability to show everything within a 1 click from the parent view
* In wallet, in the advance settings we might have the PSBT upload section, instead of having to complete the PSBT form by hand
  * PSBT work to be done once Dennis has pushed a first draft PR
* Optimizing the mobile view of the transaction list will become needed, since not a lot of live has been given to mobile views
* Seeing the actions besides the filter makes sense, all is coming together and will be a good follow up now that 1.4.X is out.

----

**Early Dashboard Wireframe Discussion**
* Dstrukt presents early Figma mockups
  * Includes a "what's new", set either to the right side or top
* We need to define what dashboard items we would like to add in it in the beginning
  * What would be beneficial for the user to see?
  * Surfacing key parts of what is happening in the store
* In this initial release, we can highlight which invoice needs an action
  * High-level split-second graphical view above the fold, with a CTA for "XYZ invoices require your attention" wording for important highlights
  * Under the fold we can then expand on more granular and detailed information
* Dstrukt to work on the v1 draft and mock-ups

----

For 1.5 the big improvements will be wallet and dashboard + minor fixes
Checkout invoice view will be a none-prioritized item that can still start to be mocked-up while Dennis is in the Walletverse.
